.github/workflows/style.yml

```diff
@@ -24,6 +24,20 @@ jobs:
           submodules: recursive
           ref: '${{ inputs.myCommit }}'
 
+      - name: Update and dependencies
+        run: |
+          sudo apt-get update -y
+          sudo apt-get install build-essential -y
```

Comment:
Is this required? All the build-essential deps should be installed on GH worker.

Reply:
I can try without them and see if the errors still pop up

Reply:
Yup! It works without them as well 😃